src/moscot/base/output.py

```diff
@@ -138,14 +135,14 @@ def as_linear_operator(self, scale_by_marginals: bool = False) -> LinearOperator
         Parameters
         ----------
         scale_by_marginals
-            Whether to scale by marginals.
+            Whether to scale by :term:`marginals`.
 
         Returns
         -------
         The :attr:`transport_matrix` as a linear operator.
         """
-        push = partial(self.push, scale_by_marginals=scale_by_marginals)
-        pull = partial(self.pull, scale_by_marginals=scale_by_marginals)
+        push = functools.partial(self.push, scale_by_marginals=scale_by_marginals)
```

Comment:
why is this? I notice everywhere that you don't import the top-level object but always refer to the module. This changed from before, could you elaborate on this practice? As in, I understand why in other places but not here.

Reply:
We decided to always use the module